stats/parsing-stats/lang/cpp/projects.txt

```diff
@@ -5,8 +5,9 @@
 https://github.com/tensorflow/tensorflow
 https://github.com/electron/electron
 https://github.com/microsoft/terminal
-https://github.com/apple/swift
-https://github.com/opencv/opencv
+# brandon: These two induce a SIGSEGV on my machine.
```

Comment:
that's not good ... not good at all ... just like for Dart we need to investigate those.

Reply:
please create an issue so that we remember to fix this.